examples/52-delete-field-in-content-type-with-editor-layout.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function (migration) {
+  const contentTypeWithEditorLayout1 = migration.editContentType('page')
```

Comment:
Do we need these `1` and `2` in the names?

Reply:
no we don't, I just used that to debug why the migration wasn't running properly, thank you for noticing!